docker-compose.yml

```diff
@@ -20,39 +20,38 @@
 web:
   build: .
   command: inveniomanage runserver -h 0.0.0.0 -p 5000
-  expose:
-   - "5000"
+  ports:
+    - "5000:5000"
   environment:
-   - CACHE_REDIS_HOST=redis
-   - CFG_DATABASE_HOST=mysql
-   - INVENIO_ENABLE_ENV=CACHE_REDIS_HOST,CFG_DATABASE_HOST
+    - CACHE_REDIS_HOST=redis
```

Comment:
should it be `cache` - see https://github.com/jirikuncar/invenio/pull/317/files#diff-4e5e90c6228fd48698d074241c2ba760R33